content/chapter 3/3.3-sync.md

```
@@ -412,28 +421,36 @@ for _, task := range tasks {
+	defer func() { <-sem }() // آزاد کردن اسلات (ایمن‌تر)
 	fmt.Printf("Processing %d\n", id)
 	time.Sleep(time.Second)
-	<-sem                      // آزاد کردن اسلات
 }
 
 func main() {
 	var wg sync.WaitGroup
 	sem := make(chan struct{}, 3) // حداکثر 3 goroutine همزمان
 
-	for i := 0; i < 10; i++ {
+	for i := range 10 {
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)